code/src/Components/ThoughtsList.js

```diff
@@ -0,0 +1,73 @@
+import React, { useEffect, useState } from 'react';
+import moment from 'moment';
+import { Likeathought } from 'Components/Likeathought';
+import 'styles/happythoughts.css';
+
+export const ThoughtsList = () => {
+  const THOUGHTS_URL = "https://happy-thoughts-technigo.herokuapp.com/thoughts";
+  const [thoughts, setThoughts] = useState([]);
+  // Use [] in the useState since it is an array that we get in the get-request
+  const [addThought] = useState("");
```

Comment:
And where is the state handler for update?

Reply:
I have done a new push now. It works fine without state handler, or how shall I create one? :) @EmOlsz

Reply:
Yep, I see, but there is not logical reason to have this `addThought` property in state if you never update it :D
In state we should have only values that are going to somehow change after some exact actions of ours.
In this example however you initalise you `addThought` as empty string and never update it. Moreover, in line 35 of this file you added `addThought` into dependency array if `useEffect()`, but since your state property will never change its value, your `useEffect()` will not be executed

Reply:
Now I removed it @EmOlsz So I hope I did right now.. :)